src/schemas/json/typedoc.json

```diff
@@ -110,14 +113,12 @@
     "plugin": {
       "description": "Specify the npm plugins that should be loaded. Omit to load all installed plugins.",
       "oneOf": [
-        {
-          "enum": ["none"]
-        },
         {
           "items": {
             "type": "string"
           },
-          "type": "array"
+          "type": "array",
+          "default": ["none"]
```

Comment:
This isn't quite right, if I'm understanding the schema correctly. This defaults to an empty array, and if it is an empty array TypeDoc will load everything.

Reply:
In VSCode, using the schema results in this auto-suggestion: ![VSCode Plugin Typedoc Schema Suggestion](https://shared.benc.io/screencaps/2019/05/f2f3b5fb-5281-4380-8983-eb4606e82893.png)

I think that matches the desired behavior?

Reply:
I guess that works, I thought the default key indicated the value that was set if it was not provided, but if it is intended as primarily an autocomplete help, this value makes sense. Someone on the team here is probably the most qualified to determine what it should be set to.

Reply:
Hm it's hard to determine. In prettier, the schema for `printWidth` is set to 80 and that's also what the `printWidth` is if no value is specified. This would be a little different.